src/main/java/com/atlassian/bitbucket/jenkins/internal/credentials/JenkinsToBitbucketCredentialsImpl.java

```diff
@@ -18,12 +18,12 @@
 
     @Override
     public BitbucketCredentials toBitbucketCredentials(String credentialId) {
-        Credentials credentials = CredentialUtils.getCredentials(credentialId);
-        return credentials != null ? toBitbucketCredentials(credentials) : ANONYMOUS_CREDENTIALS;
+        return CredentialUtils.getCredentials(credentialId)
+                .map(this::toBitbucketCredentials).orElse(ANONYMOUS_CREDENTIALS);
```

Comment:
The orElse is because getCredentials is nullable, and I preferred this to an `orElse(null)` call on that method. Might keep this as is if that's okay.